src/index.test.ts

```diff
@@ -51,6 +51,72 @@ describe("traverse", () => {
       expect(mockMutation).toHaveBeenCalledTimes(3);
     });
 
+    it("traverses additionalProperties as boolean", () => {
+      const testSchema: any = {
+        additionalProperties: true
+      };
+      const mockMutation = jest.fn((mockS) => mockS);
+
+      traverse(testSchema, mockMutation);
+
+      expect(mockMutation).toHaveBeenCalledWith(testSchema.additionalProperties);
+      expect(mockMutation).toHaveBeenCalledWith(testSchema);
+      expect(mockMutation).toHaveBeenCalledTimes(2);
+    });
+
+    it("traverses additionalProperties as schema", () => {
+      const testSchema: any = {
+        additionalProperties: {
```

Comment:
While i know that the type is implied when there is a properties field, I condemn it as bad practice, and would propose that we make type required in our own json schema meta-schema.
```suggestion
        additionalProperties: {
          type: "object",
```